CONTRIBUTING.md

```diff
@@ -47,6 +48,17 @@ rule_files:
   - more-rules.yaml
 ```
 
+This repository contains Go unit tests. Test files are located in the `o11y/exporters` directory with `_tests.go` file names.
+
+To run all the Go unit tests in the repository, execute the following command from the project's root directory `go clean -testcache && go test ./...` a similar output is expected:
```

Comment:
I'm not sure what `go clean -testcache` does exactly, but caching exists for a reason and I wouldn't like contributors to clean it every time as a habit.
Eventually, it might mean significantly longer execution times. I don't want all contributors to just use unless it's really needed.

Reply:
Makes sense. I updated it with just `go test ./...`